src/golang/lib/repos/sqlite/workflow.go

```diff
@@ -0,0 +1,220 @@
+package sqlite
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/aqueducthq/aqueduct/lib/collections/utils"
+	"github.com/aqueducthq/aqueduct/lib/database"
+	"github.com/aqueducthq/aqueduct/lib/models"
+	"github.com/aqueducthq/aqueduct/lib/models/shared"
+	"github.com/aqueducthq/aqueduct/lib/models/views"
+	"github.com/aqueducthq/aqueduct/lib/repos"
+	"github.com/dropbox/godropbox/errors"
+	"github.com/google/uuid"
+)
+
+type workflowRepo struct {
+	workflowReader
+	workflowWriter
+}
+
+type workflowReader struct{}
+
+type workflowWriter struct{}
+
+func NewWorklowRepo() repos.Workflow {
+	return &workflowRepo{
+		workflowReader: workflowReader{},
+		workflowWriter: workflowWriter{},
+	}
+}
+
+func (*workflowReader) Exists(ctx context.Context, id uuid.UUID, db database.Database) (bool, error) {
+	return utils.IdExistsInTable(ctx, id, models.WorkflowTable, db)
+}
+
+func (*workflowReader) Get(ctx context.Context, id uuid.UUID, db database.Database) (*models.Workflow, error) {
+	query := fmt.Sprintf(
+		`SELECT %s FROM workflow WHERE id = $1;`,
+		models.WorkflowCols(),
+	)
+	args := []interface{}{id}
+
+	return getWorkflow(ctx, db, query, args...)
+}
+
+func (*workflowReader) GetByOwnerAndName(ctx context.Context, ownerID uuid.UUID, name string, db database.Database) (*models.Workflow, error) {
+	query := fmt.Sprintf(
+		`SELECT %s FROM workflow WHERE user_id = $1 and name = $2;`,
+		models.WorkflowCols(),
+	)
+	args := []interface{}{ownerID, name}
+
+	return getWorkflow(ctx, db, query, args...)
+}
+
+func (*workflowReader) GetLatestStatusesByOrg(ctx context.Context, orgID uuid.UUID, db database.Database) ([]views.LatestWorkflowStatus, error) {
+	// Get workflow metadata (id, name, description, creation time, last run time, and last run status)
+	// for all workflows whose `organization_id` is `organizationId` ordered by when the workflow was created.
+	// Get the last run DAG by getting the max created_at timestamp for all workflow DAGs associated with each
+	// workflow in the organization.
+
+	// We want to return 1 row for each workflow, so we use a LEFT JOIN between the workflow_dag
+	// and workflow_dag_result tables. A LEFT JOIN outputs all rows in the left table even if there
+	// is no match with a row in the right table. If there is no match, the columns of the right table
+	// are NULL.
+	// This means that `last_run_at` and `status` in the query output can be NULL.
+	query := `
+		WITH workflow_results AS
```

Comment:
@eunice-chan For this query all I did was add indenting for each clause, ex `SELECT`, `FROM`, etc. I did not improve any of the table name aliasing, as that can be done as part of a later PR. Let me know if you have any other ideas for improving readability for SQL queries.